Makefile

```diff
@@ -30,6 +30,15 @@ PYTHON_TARGET := 2.7
 REQUIREMENTS := test-requirements.txt requirements.txt
 PIP_OPTIONS := $(ST2_PIP_OPTIONS)
 
+NODE_INDEX ?= 0
+NODE_TOTAL ?= 1
+
+ifeq ($(shell test $(NODE_TOTAL) -gt 1; echo $$?),0)
+  TEST_DB_NAME := st2-test-$(NODE_INDEX)
```

Comment:
Experimenting if we can perhaps run tests in parallel inside a single vm / container, but yeah, it's probably too complicated (there are other shared resources) and not worth the effort.